psydac/feec/multipatch/examples_nc/h1_source_pbms_nc.py

```diff
@@ -0,0 +1,279 @@
+# coding: utf-8
```

Comment:
Please add a short module docstring, which will be visible in the documentation.